### PR TITLE
docs(configuration): document webpackIgnore for commonjs

### PR DESCRIPTION
**Pull request by chenxsan**
closes https://github.com/webpack/webpack.js.org/issues/4470

### CODE REVIEW

File: src/content/api/module-methods.mdx [hunk 1]
```diff
@@ -172,7 +172,7 @@ T> Note that `webpackInclude` and `webpackExclude` options do not interfere with
 The goal of CommonJS is to specify an ecosystem for JavaScript outside the browser. The following CommonJS methods are supported by webpack:
 
 
-### `require`
+### require
```
Comment (montogeek) [inline src/content/api/module-methods.mdx:175]: Don't see the need to make it MDX.

File: src/content/configuration/module.mdx [hunk 2]
```diff
@@ -107,6 +107,51 @@ module.exports = {
...
+};
+```
+
+It's allowed to configure those options in [`Rule.parser`](/configuration/module/#ruleparser) as well to target specific modules.
+
+#### module.parser.javascript.commonjsMagicComments
+
+Enable [magic comments](/api/module-methods/#magic-comments) support for CommonJS.
+
+- Type: `boolean`
```
Comment (montogeek) [inline src/content/configuration/module.mdx:134]: We usually put the types below the title, Why this change?

**Review by montogeek — APPROVED** [review]

File: src/content/api/module-methods.mdx (hunk 1 above)
Comment (chenxsan) [reply src/content/api/module-methods.mdx:175]: See this pull request https://github.com/webpack/webpack.js.org/pull/4412, with mdx we can customize the link with `Link` from `react-router` which would improve the page performance a lot because users don't have to load the full page again clicking on a link.

Comment (montogeek) [reply src/content/api/module-methods.mdx:175]: Right :)

File: src/content/configuration/module.mdx (hunk 2 above)
Comment (chenxsan) [reply src/content/configuration/module.mdx:134]: It's explicit, and I believe it's much readable. Moreover, with explicit format we could possibly customize those complex types with [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) element in the future.
